mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -686,7 +687,7 @@ export function PortfolioScreen({ navigation }: Props): JSX.Element {
         {activeButtonGroup === ButtonGroupTabKey.AllTokens && (
           <DFIBalanceCard denominationCurrency={denominationCurrency} />
         )}
-        {!hasFetchedToken ? (
+        {!(hasFetchedToken && hasFetchedEvmTokens) ? (
```

Comment:
lets add if domain is EVM then wait for `hasFetchedEvmTokens` and otherwise wait for `hasFetchedToken`